commands/prune.go

```diff
@@ -148,10 +149,14 @@ func pruneCmd(dockerCli command.Cli, rootOpts *rootOptions) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVarP(&options.all, "all", "a", false, "Include internal/frontend images")
 	flags.Var(&options.filter, "filter", `Provide filter values (e.g., "until=24h")`)
-	flags.Var(&options.keepStorage, "keep-storage", "Amount of disk space to keep for cache")
+	flags.Var(&options.minStorage, "min-storage", "Minimum amount of disk space to keep for cache")
```

Comment:
This part needs updates based on https://github.com/moby/buildkit/pull/5359 

Not a blocker but need to make sure we don't ship this version.